README.md

```diff
@@ -52,28 +50,38 @@ Tested with the following:
 
 * `letsencrypt_domain` - Domain the certificate is for.
 * `letsencrypt_email` - Your email as certificate owner.
+* `letsencrypt_tos_sha256` - Provide the SHA256 hash of the latest Terms-of-Service you agreed to
```

Comment:
LetsEncrypt TOS handling can be discussed in a separate issue. To keep the previous behavior of `--agree-terms` being enabled by default, we can just set the default to the hash of the current terms of service document and make a new release whenever this document was updated.